lib/team_api/joiner.rb

```diff
@@ -101,37 +108,20 @@ def initialize(site)
       @public_mode = site.config['public']
     end
 
+    # Jekyll seems to be removing non-alpha characters from the team member
+    # names, causing the TeamIndexer to not be able to find any team member
+    # matches. This changes all of the team member keys back to what we expect.
+    def restructure_team_data!
+      site.data['team'].keys.each do |key|
+        site.data['team'][site.data['team'][key]['name']] = site.data['team'].delete(key)
+      end unless site.data['team'].nil? || !site.data['team'].respond_to?(:keys)
```

Comment:
Hey, I like `unless`... unless it's used in a context that's somewhat confusing. :-P There are definitely places where it's useful, but agree the more traditional `if` condition is clearer here.